galsim/galaxy_sample.py

```diff
@@ -270,18 +260,12 @@ def _apply_exclusion(self, exclusion_level, min_hlr=0, max_hlr=0, min_flux=0, ma
             # will first set that column to 1.e-5.  We choose a sample-dependent mask ratio cut,
             # since this depends on the peak object flux, which will differ for the two samples
             # (and we can't really cut on this for arbitrary user-defined samples).
-            if self.use_sample == "23.5":
-                cut_ratio = 0.2
-                sn_limit = 20.0
-            else:
-                cut_ratio = 0.8
-                sn_limit = 12.0
             div_val = self.selection_cat['peak_image_pixel_count']
             div_val[div_val == 0.] = 1.e-5
-            mask &= ( (self.selection_cat['sn_ellip_gauss'] >= sn_limit) &
+            mask &= ( (self.selection_cat['sn_ellip_gauss'] >= self.sn_limit) &
                       ((self.selection_cat['min_mask_dist_pixels'] > 11.0) |
```

Comment:
This still seems to be hardcoded to 11, whereas the other parameters are now quantities that can be passed in; was this distinction deliberate?

Reply:
No.  Just an oversight.  Thanks.  I made it a parameter now.